step.yml

```diff
@@ -111,6 +111,23 @@ inputs:
         - "yes"
         - "no"
       is_required: true
+  - disable_index_while_building: "yes"
+    opts:
+      title: "Disable index while building"
```

Comment:
`Disable indexing during the build` would sound better I guess.

Reply:
Fixed in:
https://github.com/bitrise-steplib/steps-xcode-test/pull/111/commits/dbcded918e7354f29d6051ce30f3376994436422